packages/firestore/README.md

```diff
@@ -10,7 +10,7 @@ For Flutter plugins for other Firebase products, see [FlutterFire.md](https://gi
 
 To use this plugin:
 
-1. Using the [Firebase Console](http://console.firebase.google.com/), add an Android app to your project: Follow the assistant, download the generated google-services.json file and place it inside android/app. Next, modify the android/build.gradle file and the android/app/build.gradle file to add the Google services plugin as described by the Firebase assistant.
+1. Using the [Firebase Console](http://console.firebase.google.com/), add an Android app to your project: Follow the assistant, download the generated google-services.json file and place it inside android/app. Next, modify the android/build.gradle file and the android/app/build.gradle file to add the Google services plugin as described by the Firebase assistant. Ensure that your `android/build.gradle` file contains the `maven.google.com` as [described here](https://firebase.google.com/docs/android/setup#add_the_sdk).
```

Comment:
non-actionable comment: any flutter app that was created within the last 4 month already includes this one per default (after creating it with `flutter create`).